ddl/ingest/backend_mgr.go

```diff
@@ -121,7 +122,7 @@ func createLocalBackend(ctx context.Context, cfg *Config) (*local.Backend, error
 	regionSizeGetter := &local.TableRegionSizeGetterImpl{
 		DB: nil,
 	}
-	backendConfig := local.NewBackendConfig(cfg.Lightning, int(LitRLimit), cfg.KeyspaceName)
+	backendConfig := local.NewBackendConfig(cfg.Lightning, int(LitRLimit), cfg.KeyspaceName, resourcegroup.DefaultResourceGroupName)
```

Comment:
I'm not sure which group we should use here

Reply:
Oh, it must be current_group as well, my fault!

Reply:
After checking the code, it will be supported after the ddl resource group is supported. For now, I think we can ignore by `""`.